tests/pc/test_po_cleanup.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import logging
+from tests.common.utilities import wait_until
+from tests.common import config_reload
+
+
+pytestmark = [
+    pytest.mark.topology('any'),
+]
+
+@pytest.fixture(autouse=True)
+def ignore_expected_loganalyzer_exceptions(duthost, loganalyzer):
+    """
+        Ignore expected failures logs during test execution.
+
+        LAG tests are triggering following syncd complaints but the don't cause
+        harm to DUT.
+
+        Args:
+            duthost: DUT fixture
+            loganalyzer: Loganalyzer utility fixture
+    """
+    # when loganalyzer is disabled, the object could be None
+    if loganalyzer:
+        ignoreRegex = [
+            ".*ERR swss#orchagent: :- removeLag.*",
+        ]
+        loganalyzer.ignore_regex.extend(ignoreRegex)
+        expectRegex = [
+            ".*teamd#teammgrd: :- cleanTeamProcesses.*",
+            ".*teamd#teamsyncd: :- cleanTeamSync.*"
+        ]
+        loganalyzer.expect_regex.extend(expectRegex)
+
+    yield
+
+
+def check_kernel_po_interface_cleaned(duthost):
+    res = duthost.shell("ip link show | grep -c PortChannel | cat")["stdout_lines"][0].decode("utf-8")
```

Comment:
What does this pipe to cat do?

Reply:
Are you trying to mute the return errorcode? you can use command_output = self.shell(cmd, module_ignore_errors=True) to ignore the command error code instead.

Reply:
@yxieca yes, that was the reason for using cat. with output as "0" ansible module restrn error.
I will update to use module_ignore_errors as True

Reply:
@yxieca Done